### DISCUSSION TITLE
How to correctly handle unit scale factor(s)?

### DISCUSSION THREAD

**Discussion by Lucipetus** [assimp/assimp#4018] [Q&A] [answered]
I have some FBXs that when imported into Unity, appear to scale correctly, but with ASSIMP some appears to be 100 times bigger than others.

I noticed the smaller model seems to be scaled in meters, and it's got the following properties:
UnitScaleFactor: 100
OriginalUnitScaleFactor: 100

The 100 times bigger one:
UnitScaleFactor: 1
OriginalUnitScaleFactor: 1

I found an explanation on both factors here: 
https://forums.autodesk.com/t5/3ds-max-programming/maxscripting-the-fbx-exporter/td-p/9984666
But I still can't figure out what they really mean. The post says FBX data is in centimeters. Does it mean it's in centimeters before or after the scale factor (which or both?) is applied?

// major edit
I just found out the reason I couldn't get UnitScaleFactor was that I mistakenly thought it was double, and that I can use aiProcess_GlobalScale to achieve the right scaling, and it only involves UnitScaleFactor. 
So OriginalUnitScaleFactor doesn't matter after all, and FBX coordinates are in centimeters only after unit scale factor is applied to them?
Am I right?

**Comment by kimkulling** [comment, marked as answer]
Thanks a lot for sharing your answer!
